packages/fiori-generator-shared/src/npm-package-scripts/getPackageScripts.ts

```diff
@@ -105,26 +88,25 @@ function getVariantPreviewAppScript(sapClient?: string): string {
  * @param options.localOnly no server available
  * @param options.addMock add a script for using the mockserver
  * @param options.addTest add a script for executing OPA tests
- * @param options.sapClient SAP client required for connecting to the backend
  * @param options.flpAppId local FLP id
  * @param options.startFile path that should be opened with the start script
  * @param options.localStartFile path that should be opend with the start-local script
  * @param options.generateIndex exclude the start-noflp script
+ * @param options.addSearchParams boolean to determine whether to add search params to the start command
```

Comment:
Same comment, why would you set this? In this context it means add very specific search params.